brickstrap-paths.sh

```diff
@@ -0,0 +1,445 @@
+#!/bin/bash
+#
+# This file is part of brickstrap.
```

Comment:
I like to have a copyright statement at the beginning of every source file. You should add yourself to the copyright too.